erpnext/patches/v13_0/disable_ksa_print_format_for_others.py

```diff
@@ -3,7 +3,7 @@
 
 import frappe
 
-from erpnext.regional.saudi_arabia.setup import add_print_formats
+from ksa.install import add_print_formats
```

Comment:
All the patches should also be moved to the custom app, there should be no dependency on any other app